lib/wallaby/browser.ex

```diff
@@ -392,12 +392,37 @@ defmodule Wallaby.Browser do
   @doc """
   Sets the value of an element.
   """
-  @spec set_value(element, any()) :: element
+  @spec set_value(parent, Query.t, Element.value) :: parent
+#  @spec set_value(element, any()) :: element
+
+#  def set_value(element, value) do
+#    IO.warn "set_value/2 has been deprecated. Please use Element.set_value/2"
+#
+#    Element.set_value(element, value)
+#  end
```

Comment:
I think we want to keep this in the `Browser` module since it accepts queries. We will want to remove the commented out code.